loadtest/user/userentity/actions.go

```diff
@@ -156,6 +158,21 @@ func (ue *UserEntity) CreateDirectChannel(otherUserId string) (string, error) {
 
 	return channel.Id, nil
 }
+func (ue *UserEntity) RemoveUserFromChannel(channelId, userId string) error {
+	ok, resp := ue.client.RemoveUserFromChannel(channelId, userId)
+	if !ok {
+		return errors.New("Cant delete the user in the server")
+	}
+	if resp.Error != nil {
+		return resp.Error
+	}
+	err := ue.store.RemoveChannelMember(channelId, userId)
+	if err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
We can probably simplify this and directly return `ue.store.RemoveChannelMember(channelId, userId)`